eng/common/testproxy/test-proxy-tool.yml

```diff
@@ -21,7 +21,7 @@ steps:
 
   - pwsh: |
       Start-Process $(Build.BinariesDirectory)/test-proxy/test-proxy.exe `
-        -ArgumentList "--storage-location '${{ parameters.rootFolder }}'" `
+        -ArgumentList "--storage-location ${{ parameters.rootFolder }}" `
         -NoNewWindow -PassThru -RedirectStandardOutput $(Build.SourcesDirectory)/test-proxy.log
     displayName: 'Run the testproxy - windows'
```

Comment:
@scbedd for reference

Reply:
Analyze step will be fixed once this fix goes into the tools repo. https://github.com/Azure/azure-sdk-tools/pull/2290